_stbt/match.py

```diff
@@ -452,10 +458,34 @@ def _find_matches(image, template, match_parameters, imglog):
     if template.dtype != numpy.uint8:
         raise ValueError("Template image must be 8-bits per channel")
 
+    structure_weighting = numpy.clip(match_parameters.structure_weighting, 0, 1)
+    image_weighting = 1 - structure_weighting
+
+    images = []
+    templates = []
+    weights = []
+
+    if structure_weighting > 0.05:
+        images.append(cv2.Sobel(image, cv2.CV_32F, 1, 0, ksize=3))
+        templates.append(cv2.Sobel(template, cv2.CV_32F, 1, 0, ksize=3))
+        weights.append(structure_weighting * 0.5)
+
+        images.append(cv2.Sobel(image, cv2.CV_32F, 0, 1, ksize=3))
+        templates.append(cv2.Sobel(template, cv2.CV_32F, 0, 1, ksize=3))
+        weights.append(structure_weighting * 0.5)
```

Comment:
Why do the x & y Sobel separately?

Reply:
I see. Specifying `xorder=1, yorder=1` gives diagonal edges, which isn't what we want.